labs/lab18-helm-charts/README.md

```diff
@@ -0,0 +1,435 @@
+
+#### Adding kots-app.yaml
+
+The first Custom Resource we'll create is [Application](https://docs.replicated.com/reference/custom-resource-application). This manifest includes details about the application we created.
+
+``` yaml
+#kots-app.yaml
+apiVersion: kots.io/v1beta1
+kind: Application
+metadata:
+  name: helm-wordpress
+spec:
+  title: Helm Wordpress
+  icon: https://upload.wikimedia.org/wikipedia/commons/9/93/Wordpress_Blue_logo.png
+  statusInformers:
+    - deployment/wordpress
+
+```
+
+#### Adding wordpress.yaml
+
+The second Custom Resource we'll create is [Helm Chart](https://docs.replicated.com/reference/custom-resource-helmchart). This file has information about the Wordpress Helm Chart. This file is specific to the Helm Chart, so if your application contains multiple, separate charts, you will need a manifest for each Helm Chart. You do not need to define a `Helm Chart` manifest for any chart included as a sub-chart.
+
+```yaml
+#wordpress.yaml
+apiVersion: kots.io/v1beta1
+kind: HelmChart
+metadata:
+  name: wordpress
+spec:
+  # chart identifies a matching chart from a .tgz
+  chart:
+    name: wordpress
+    chartVersion: "14.0.7"
+  # helmVersion identifies the Helm Version used to render the chart. Default is v2.
+  helmVersion: v3
+
+  # useHelmInstall identifies whether this Helm chart will use the
+  # Replicated Helm installation (false) or native Helm installation (true). Default is false.
+  # Native Helm installations are only available for Helm v3 charts.
+  useHelmInstall: true
+
+  # weight determines the order that charts with "useHelmInstall: true" are applied, with lower weights first.
+  weight: 0
+
+  # values are used in the customer environment, as a pre-render step
+  # these values will be supplied to helm template
+  values: {}
+```
+
+Before we create our release, let's initialize git. We will use a command line option that uses Git metadata to determine the version label, channel to promote to and more.
+
+```bash
+    $ git init
+    $ git add .
+    $ git commit -m "this is my first commit"
+```
+Now we are ready to create our first release:
+
+```bash
+    $ replicated release create --auto --promote lab18-helm-charts -y
+```
+
+## Deploy the Application
+
+We now have something we can deploy! As part of this lab, you were provided with the IP address of a Virtual Machine. Log in using `ssh kots@{IP-Address}` using the IP address provided. The password will be provided during this lab.
+
+Once you are on the terminal, copy the "Embedded Cluster" install command from the channel (`lab18-helm-charts`). It should look like
+```
+curl -sSL https://k8s.kurl.sh/YOUR-APP-NAME-lab18-helm-charts | sudo bash
+```
+
+Go through the installation steps in kotsadm to install the application. You should have already completed [Lab0: Hello World](https://github.com/replicatedhq/kots-field-labs/blob/main/labs/lab00-hello-world/README.md), so this should be familiar.
```

Comment:
```suggestion
Once this kicks off, consider grabbing a cup of coffee as the Kubernetes installation can take 5-10 minutes 🙂

Once the install is complete, follow the IP provided and continue with the installation steps in the App Manager UI to upload a license install the application. Since you have already completed [Lab0: Hello World](https://github.com/replicatedhq/kots-field-labs/blob/main/labs/lab00-hello-world/README.md), this should be pretty familiar.
```